sqlmesh/core/renderer.py

```diff
@@ -527,6 +527,11 @@ def _optimize_query(self, query: exp.Subqueryable, all_deps: t.Set[str]) -> exp.
 
 @contextmanager
 def _normalize_and_quote(query: E, dialect: str, default_catalog: t.Optional[str]) -> t.Iterator[E]:
+    if isinstance(query, (exp.Command, exp.AlterTable)) and default_catalog:
```

Comment:
i don't think this is the right fix. shouldn't we just fix qualify_tables in sqlglot to work with alter tables?

Reply:
Also `exp.Command` parses everything after as a literal, I just realized
Is there a `@default_catalog` macro var? If not, I think that might be the move here. A 2 pronged approach. Fix `qualify_tables` in sqlglot, and use `@default_catalog` for users trying to run `GRANT`s while not hardcoding a catalog
@tobymao

Reply:
makes sense, do you want to just make a PR to ensure default_catalog is exposed as a macro var? we can handle qualify_tables cc @georgesittas

Reply:
Sure. Also Im on the default tables thing now, just give me 2 mins. 😃 add test